webapp/builder/logic.py

```diff
@@ -0,0 +1,66 @@
+import os
+import secrets
+import urllib.parse
+
+import requests
```

Comment:
`from webapp import api` here is our custom request. Where you can do: `api.requests`

and could you register the Session that you use in talisker: `talisker.requests.configure(YOUR_SESSION)` this will allow us to get more info iin the logs about the api calls that we are making